vpr/src/base/netlist_walker.h

```diff
@@ -25,16 +25,24 @@ class NetlistVisitor {
         virtual ~NetlistVisitor() = default;
         void start() { start_impl(); }
         void visit_top(const char* top_level_name) { visit_top_impl(top_level_name); }
-        void visit_clb(const t_pb* clb) { visit_clb_impl(clb); }
+        void visit_clb(ClusterBlockId blk_id, const t_pb* clb) { visit_clb_impl(blk_id, clb); }
         void visit_atom(const t_pb* atom) { visit_atom_impl(atom); }
+        void visit_open(const t_pb* atom) { visit_open_impl(atom); }
```

Comment:
The visit_open code is required because of route through LUTs.  So calling it visit_unused_atom is wrong as well, because the atom is in use.  The .net format lists route through LUTs are name="open", which becomes atom->name == nullptr.

Reply:
Interesting, I wouldn't have gathered that from the name. Perhaps `visit_route_through()` would be more descriptive?

Reply:
Sure, changed.